src/pyramid/config/security.py

```diff
@@ -60,8 +60,8 @@ def set_authentication_policy(self, policy):
         """
         .. deprecated:: 2.0
 
-            Authentication policies have been replaced by
-            security policies.  See :ref:`upgrading_auth` for more information.
+            Authentication policies have been replaced by security policies.
+            See :ref:`upgrading_auth_20` for more information.
```

Comment:
```suggestion
            See :ref:`upgrading_auth` for more information.
```

There is no target `upgrading_auth_20`, but `upgrading_auth` is used. Perhaps we should be explicit about the version number throughout both the source and docs and append `_20`?